### DISCUSSION TITLE
Addding sorting parameters for pagination

### DISCUSSION THREAD

**Discussion by s-rigaud** [uriyyo/fastapi-pagination#606] [Ideas]
Hello @uriyyo 👋 

I'm interested in adding this package to one of my personal project but I also need to integrate sorting logic to my endpoints. I would like to be able to automatically infer endpoint results and OpenAPI schema to sort the result.
Do you think I should integrate this new feature in your project and create my own with sorting logic (both project would then be really similar) ?

**Comment by uriyyo** [comment]
Hi @s-rigaud,

I guess it would be better to create your own sorting logic, or you can try to use my other package for sorting and filtering - https://github.com/uriyyo/fastapi-filters
